Reply/app/src/main/java/com/example/reply/ui/navigation/ReplyNavigationActions.kt

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
```

Comment:
Comment for this entire class - why treat the navigation actions as vals instead of functions? Especially when naming them as actions "navigateTo..."

Reply:
Thanks, Adopted this approach as well.